vitest.config.mts

```diff
@@ -16,6 +16,7 @@ function generateConfig(): Promise<UserConfigExport> {
             '**/cypress/**',
             '**/.{idea,git,cache,output,temp}/**',
             '**/{karma,rollup,webpack,vite,vitest,jest,ava,babel,nyc,cypress,tsup,build}.config.*',
+            'arches_lingo/constants.ts',
```

Comment:
I'll revert this: I was misunderstanding how/when this file got covered in whole or in part.